src/Jackett.Common/Indexers/TorrentHeaven.cs

```diff
@@ -32,7 +32,7 @@ private new ConfigurationDataCaptchaLogin configData
         public TorrentHeaven(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
             : base(name: "TorrentHeaven",
                    description: "A German general tracker.",
-                   link: "https://torrentheaven.myfqdn.info/",
+                   link: "https://newheaven.nl/",
```

Comment:
OK then.
so please add the old domain to a `LegacySiteLinks` block, so that existing user's configs get their old link automatically replaced by the new site link.
see the following as an example:
https://github.com/Jackett/Jackett/blob/060972475f13ffe59fcc09c51ffe91a547a29029/src/Jackett.Common/Indexers/yts.cs#L20-L22

and please add the new CA thumbprint  so that Jackett does not get a https invalid CA error.
I suggest converting:  https://github.com/Jackett/Jackett/blob/23e1d4baa2a3aa8378a53055342cff0daa020bd4/src/Jackett.Common/Indexers/TorrentHeaven.cs#L24
into an string array `certificateHashs` and include the new CA thumbprint `66096db5fd0107e4ffbaf5ec8378eb235cada909`
and then modify
https://github.com/Jackett/Jackett/blob/23e1d4baa2a3aa8378a53055342cff0daa020bd4/src/Jackett.Common/Indexers/TorrentHeaven.cs#L95
by using this example as a model
https://github.com/Jackett/Jackett/blob/060972475f13ffe59fcc09c51ffe91a547a29029/src/Jackett.Common/Indexers/CardigannIndexer.cs#L191-L192
Thanks.